img/logo.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="199" height="93" viewBox="0 0 199 93" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Możesz użyć narzędzia: https://jakearchibald.github.io/svgomg/ w celu otymalizacji takiego pliku SVG.

W tej apce przeciągasz sobie pliczek z komputera na stronę, zaznaczasz wszystkie opcje za wyjągkiem "Remove xmlns" oraz "Remove viewbox", ustawiasz suwak "Precision" na najmniejszą wartość która nie psuje obrazka i pobierasz sobie o połowę mniejszy pliczek SVG :)

To samo możesz zrobić z plikami PNG na stronie https://squoosh.app/, albo nieco prostszej w użyciu https://tinypng.com/.

![Screenshot 2022-03-03 at 14 05 07](https://user-images.githubusercontent.com/10709214/156570393-6376d4cc-d58f-4424-9784-9ad35f1cd766.png)

Reply:
Trochę dziwna sytuacja wyszła bo ja te pliki wyeksportowałem z Figmy w formacie .svg.  Zrobiłem całe logo używając .svg. Nie wiem dlaczego kod się delikatnie różni. Kiedy wejdziesz na gałąź "navbar" i otworzysz plik index.html to tam jest  plik w formacie .svg w tagu <img>.  Wygląda to tak: <a href="javascript:void(0)"> <img src="/img/logo.svg" alt="Website logo"> </a>
 I w tym tagu właśnie zawarłem to logo. A  pliku img/flat-color-icons_generic-sorting-asc.png nie mam bo go wcześniej usunąłem ;) Nie rozumiem skąd te rozbieżności :P

Reply:
`<a href="javascript:void(0)"> <img src="/img/logo.svg" alt="Website logo"> </a>`